.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Continuous Integration
+on:
+  push:
+    branches:
+      - main
+      - candlepin-4.1-HOTFIX
+      - candlepin-4.2-HOTFIX
+
+env:
+  JAVA_DISTRIBUTION: 'temurin'
+
+jobs:
+  internationlization:
+    name: internationlization
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:38
+
+    steps:
+      - name: Mask secrets
+        shell: bash
+        run: |
+          echo "::add-mask::${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Install dependencies
+        shell: bash
+        run: dnf --setopt install_weak_deps=False install -y gettext jss git
+
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up Java 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: ${{ env.JAVA_DISTRIBUTION }}
+          java-version: 11
+
+      - name: Set up Java 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: ${{ env.JAVA_DISTRIBUTION }}
+          java-version: 17
```

Comment:
Setting up Java 11 and Java 17 because the older code lines are still on Java 11

Reply:
wouldn't it be better if each branch had its own i18n job? this way:
- the job in each branch would only care about its own branch -- no need to support multiple java versions, or multiple layouts of the sources, etc
- disabling the i18n job for a branch means only changing/removing it only in that branch, with no risks to break others

Reply:
I agree with Pino. This multi-branch juggling is an artifact of the job source code living separately than the main candlepin repo. But now that the job is part of the main repo it should live naturally in each branch

Reply:
I set the i18n jobs to run on the twice a week schedule. I also created separate jobs for each code line branch so that we can specify the setup, like Java version, that is specific to each branch.